fastn-lang/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "fastn-lang"
+version = "0.1.0"
+authors = ["Amit Upadhyay <upadhyay@gmail.com>"]
+edition = "2018"
+
+[dependencies]
+accept-language = "2"
+thiserror = "1"
+serde = "1"
+enum-iterator = "0.6.0"
+enum-iterator-derive = "0.6.0"
```

Comment:
Put version numbers in top level Cargo.toml, here write `accept-lang.workspace = true` etc.